code.py

```diff
@@ -0,0 +1,11 @@
+import json
+
+
+def lambda_handler(event, context):
+    if "body" not in event:
+        return "no body found"
```

Comment:
> [!CAUTION]
> The function returns plain string responses instead of proper AWS Lambda response objects. AWS Lambda HTTP integrations expect a dictionary with statusCode, headers, and body properties.

> [!TIP]
> Return a properly formatted response object that includes statusCode, headers and body.

```python
def lambda_handler(event, context):
    if "body" not in event:
        return {
            "statusCode": 400,
            "headers": {"Content-Type": "application/json"},
            "body": json.dumps({"error": "no body found"})
        }

    # 1. Parse GitHub Webhook
    payload = json.loads(event["body"])
    print(payload)
    return {
        "statusCode": 200,
        "headers": {"Content-Type": "application/json"},
        "body": json.dumps({"message": "ok"})
    }
```